### PR TITLE
Batch update fix gocql

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
## Fixes
Panic in gocql cluster.

```
2022/09/14 21:31:12 gocql: control unable to register events: gocql: no response to connection startup within timeout
panic: finalize block save changes failed

goroutine 89274 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc009ef3bc0, {0xc00c278300, 0x3, 0x3})
        /0chain/code/go/0chain.net/vendor/go.uber.org/zap/zapcore/entry.go:232 +0x68b
go.uber.org/zap.(*Logger).Panic(0xc000608e40, {0x18b1904, 0x22}, {0xc00c278300, 0x3, 0x3})
        /0chain/code/go/0chain.net/vendor/go.uber.org/zap/logger.go:230 +0x6c
0chain.net/chaincore/chain.(*Chain).finalizeBlock(0xc000204600, {0x19c3cd8, 0xc01c562360}, 0xc009df7860, {0x19c1050, 0x200cb80})
        /0chain/code/go/0chain.net/chaincore/chain/protocol_block.go:351 +0x17ab
0chain.net/chaincore/chain.(*Chain).finalizeBlockProcess(0xc000204600, {0x19c3cd8, 0xc01c562360}, 0xc009df7860, {0x19c1050, 0x200cb80})
        /0chain/code/go/0chain.net/chaincore/chain/worker.go:390 +0x2a9c
0chain.net/chaincore/chain.(*Chain).FinalizedBlockWorker.func1.1()
        /0chain/code/go/0chain.net/chaincore/chain/worker.go:263 +0xbe
created by 0chain.net/chaincore/chain.(*Chain).FinalizedBlockWorker.func1
        /0chain/code/go/0chain.net/chaincore/chain/worker.go:261 +0x23e
```
## Changes
- Add connectionTimeout to gocql cluster.

## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/chaincore/transaction/entity.go [hunk 1]
```diff
@@ -412,10 +412,7 @@ func (t *Transaction) GetSummary() *TransactionSummary {
 - applicable only when running in test mode and the transaction_data string contains debug keyword somewhere in it
 */
 func (t *Transaction) DebugTxn() bool {
-	if !config.Development() {
```
Comment (dabasov) [inline code/go/0chain.net/chaincore/transaction/entity.go]: this will be fixed in different PR, check this [[Logging] Add trace level](https://github.com/0chain/0chain/issues/1636)

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/chaincore/transaction/entity.go (hunk 1 above)
Comment (Kishan-Dhakan) [reply code/go/0chain.net/chaincore/transaction/entity.go]: Ah okay, I'll revert it

**Review by peterlimg — APPROVED** [review]
lgtm
